Explain downtime audit notifications (#16222)
DataDog/documentation: content/en/monitors/notify/downtimes.md

### Notify your team

> Notify your team by specifying team members or send the message to a service [integration][3]. Datadog will send notifications to the specified destinations whenever the downtime is scheduled / started / cancelled / expired. These audit notifications are useful for keeping your team in the loop about the Downtimes in your system.

Reviewer: ```suggestion
Notify your team by specifying team members or send the message to a service [integration][3]. Datadog sends notifications to the specified destinations whenever the downtime is scheduled, started, cancelled, and expired. These audit notifications allows your team to be aware of the Downtimes in your system.
```
